### ISSUE TITLE
ansible-lint 5.4.0 => AnsibleCollectionFinder has not been installed in this process

### ISSUE THREAD

**Issue by mageebr** [ansible/ansible-lint#3706]
##### Summary

Running ansible-lint 5.4.0 with a file errors with:
an AnsibleCollectionFinder has not been installed in this process

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

$ lsb_release -a 
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy

$ ansible-lint --version
ansible-lint 5.4.0 using ansible 2.15.2
```console (paste below)
ansible-lint --version
```

- ansible installation method: $apt install ansible
- ansible-lint installation method: $apt install ansible-lint

##### STEPS TO REPRODUCE

~/junk$ ansible-lint
~/junk$ echo $?
0
~/junk$ touch kdkdkdk
:~/junk$ ansible-lint 
an AnsibleCollectionFinder has not been installed in this process
mageebr@rckl-utility-vcp:~/junk$ echo $?
1
##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is happening.
Include a [minimum complete verifiable example] with:

- minimized playbook to reproduce the error
- the output of running ansible-lint including the command line used
- if you're getting a stack trace, also the output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
